CryptoPrices/Sources/MyCoin/Sources/MyCoin/MyCoinView/MyCoinView.swift

```diff
@@ -47,6 +47,9 @@ public struct MyCoinView: View {
                 await viewModel.fetchChartPricesData(id: myCoinState.id, timeFrameItem: selectedTimeFrameItem)
             }
         }
+        .refreshable {
+            await viewModel.fetchData(id: myCoinState.id, timeFrameItem: .init(timeFrame: .oneDay))
```

Comment:
It looks like we forget to update the UI of the selected filter when refreshing the chart data to `1D` filter. For example, while I am using the app, I switch the filter to `1M` and then trigger a refresh, it doesn't select back to the `1D` filter:

https://user-images.githubusercontent.com/70877098/212587847-691c44a6-691e-432c-aac6-c7cbcbbb40af.mov

And in case the refresh was unsuccessful, we might need to keep showing the same data and keep the same filter also. Please help check 🙏

Reply:
Good catch 🙏 . I forgot that we detected the change of `selectedTimeFilterItem`, so call `fetchData` there is unnecessary.
>And in case the refresh is unsuccessful, we might need to keep showing the same data and keep the same filter also. Please help check 🙏

Thank you for raising another edge case. Good point. I fixed it at  https://github.com/nimblehq/nimble-crypto-ios/pull/98/commits/444e39b98ac45e1bbaaecdaece67099f492a779e :D.